src/main/java/frc/robot/Constants.java

```diff
@@ -30,6 +30,15 @@ public class PathPlanner {
         
     }
 
+    public class Shooter {
+        public static final int INDEXER_MOTOR_ID = 1; // PLACEHOLDER
+        public static final int SHOOTER_MOTOR_ONE_ID = 2; // PLACEHOLDER
+        public static final int SHOOTER_MOTOR_TWO_ID = 3; // PLACEHOLDER
```

Comment:
lets standardize naming to `SHOOTER_MOTOR_ONE` / `TWO`

Reply:
same with indexer motors (there are 2 now)

Reply:
I am initially assigning ID's 13-16 to undertaker. Please change these

1-8 swerve
9-10 undertaker
11-12 arm
13-16 shooter

Reply:
The swerve actually needs CAN IDs 1-12, so add 4 to your subsystem can IDs.